src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +34,33 @@
               className="modal-card-title has-text-weight-medium"
```

Comment:
Consider using the classnames library for defining classes conditionally. It can make your code cleaner and easier to understand. Here is how you can use it: https://www.npmjs.com/package/classnames